tasks/init.rb

```diff
@@ -5,7 +5,17 @@
 require 'open3'
 
 def get(fact)
-  facter = (%r{mingw} =~ RUBY_PLATFORM) ? '"C:\Program Files\Puppet Labs\Puppet\bin\facter"' : '/opt/puppetlabs/puppet/bin/facter'
+  if ENV.has_key? 'Path' then
```

Comment:
Why do this rather than rely on the PATH lookup that already happens with Open3.capture3?